src/core/bitop.d

```diff
@@ -216,6 +246,24 @@ int bts(size_t* p, size_t bitnum) pure @system;
  */
 uint bswap(uint v) pure;
 
+/**
+ * Swaps bytes in an 8 byte ulong end-to-end, i.e. byte 0 becomes
+ * byte 7, byte 1 becomes byte 6, etc.
+ */
+ulong bswap(ulong v) pure
+{
+    auto sv = split64(v);
+
+    const temp = sv.lo;
+    sv.lo = bswap(sv.hi);
+    sv.hi = bswap(temp);
+
+    if (__ctfe)
+        return (cast(ulong) sv.hi << 32) | sv.lo;
+    else
+        return sv.u64;
+}
```

Comment:
Just always use the CTFE version, going through a store and different load is difficult for compilers to figure out and might result in stack usage.